src/libpriv/rpmostree-scripts.c

```diff
@@ -190,7 +190,7 @@ run_script_in_bwrap_container (int rootfs_fd,
                                /* Scripts can see a /var with compat links like alternatives */
                                "--ro-bind", "./var", "/var",
                                /* But no need to access persistent /tmp, so make it /tmp */
-                               "--bind", "/tmp", "/var/tmp",
+                               "--bind", "./tmp", "/var/tmp",
```

Comment:
See the line above...we have a ro-bind mount of `/var` so that e.g. the vagrant %post sees the
```
src/libpriv/rpmostree-postprocess.c:  if (!rpmostree_rootfs_symlink_emptydir_at (rootfs_fd, "../../usr/lib/vagrant", "var/lib/vagrant", error))
```
bit we do on the host.  We could potentially redo this stuff in the script container...was just going for a quick and correct fix here.